tests/test_check.py

```diff
@@ -61,6 +61,35 @@ def test_check_on_file():
         )
 
 
+def test_check_on_package():
+    """
+    Test verifies if `check_all()` works with package notation (dots) and also mixed with raw file paths.
```

Comment:
In general, docstring descriptions should start on the same line as the `"""`.

Also, follow the format of other descriptions, e.g. "Test inputs that consist of..."